src/patternfly/components/Banner/examples/Banner.md

```diff
@@ -51,36 +51,109 @@ cssPrefix: pf-c-banner
   disabled link</a>
 {{/banner}}
 <br>
-{{#> banner banner--modifier="pf-m-info"}}
-  Info banner with an 
+{{#> banner banner--modifier="pf-m-blue"}}
+  Blue banner with an 
   {{#> button button--modifier="pf-m-inline pf-m-link"}}
     inline link button
   {{/button}}
 {{/banner}}
 <br>
-{{#> banner banner--modifier="pf-m-warning"}}
-  Warning banner with an 
+{{#> banner banner--modifier="pf-m-gold"}}
+  Gold banner with an 
   {{#> button-link button-link--attribute='href="https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html#ARIA8-examples"' button-link--modifier="pf-m-inline pf-m-link"}}
   inline link button (anchor){{/button-link}}
 {{/banner}}
 <br>
-{{#> banner banner--modifier="pf-m-danger"}}
-  Danger banner with a 
+{{#> banner banner--modifier="pf-m-red"}}
+  Red banner with a 
   {{#> button button--modifier="pf-m-link pf-m-inline" button--attribute="disabled"}}
     disabled inline link button
   {{/button}}
 {{/banner}}
 ```
 
+### Status
+When a banner is used to convey status, it is advised to add an icon that also conveys status visually, and to provide screen reader text using ` .pf-screen-reader ` or an equivalent.
+
+```hbs
+{{#> banner}}
+  {{#> l-flex l-flex--modifier="pf-m-space-items-xs"}}
```

Comment:
Hmm... after looking at this in the PR, I think `sm` spacer would work better.

```suggestion
  {{#> l-flex l-flex--modifier="pf-m-space-items-sm"}}
```

Before:
<img width="388" alt="Screen Shot 2022-09-09 at 5 54 55 PM" src="https://user-images.githubusercontent.com/35148959/189456207-4bc53e97-eda5-45ad-b518-e1fa398a83a9.png">


After:
<img width="486" alt="Screen Shot 2022-09-09 at 5 54 46 PM" src="https://user-images.githubusercontent.com/35148959/189456217-859084cf-34e2-4efb-8b9f-0888787a7fdc.png">